src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrSettings.cs

```diff
@@ -17,8 +17,15 @@ public class NotifiarrSettings : IProviderConfig
     {
         private static readonly NotifiarrSettingsValidator Validator = new NotifiarrSettingsValidator();
 
+        public NotifiarrSettings()
+        {
+            Environment = 1;
+        }
+
         [FieldDefinition(0, Label = "API Key", Privacy = PrivacyLevel.ApiKey, HelpText = "Your API key from your profile", HelpLink = "https://notifiarr.com")]
         public string APIKey { get; set; }
+        [FieldDefinition(1, Label = "Environment", Type = FieldType.Select, SelectOptions = typeof(NotifiarrEnvironment), HelpText = "Live unless told otherwise", HelpLink = "https://notifiarr.com")]
```

Comment:
Probably should make this advanced, no?

Reply:
avoids users _easily_ changing to to dev when they don't have access / shouldn't be touching it in the first place.

this would also match the client given the dev/mode flag is not a preconfigured option (i.e. visible by default) and needs to be added explicitly by the user

Reply:
I'd agree on this one being advanced, it aligns with our usage of the flag elsewhere and keeps some out of trouble